assets/src/edit-story/panels/size.js

```diff
@@ -44,7 +45,7 @@ function SizePanel( { selectedElements, onSetProperties } ) {
 						height: typeof newWidth === 'number' && lockRatio ? newWidth / ratio : height,
 					} );
 				} }
-				postfix="px"
+				postfix={ __( 'px', 'amp' ) }
```

Comment:
Same here about using `_x()` for "px" in this file.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/4143/commits/8e29fd2233e19ebfeb7f1d1dfdda23ca3dc8fc19